### DISCUSSION TITLE
Volunteer for project maintenance

### DISCUSSION THREAD

**Discussion by seanbright** [asterisk-java/asterisk-java#611] [General]
Hi 👋🏼 

I’d like to volunteer to take over project maintenance. Please let me know what information you need from me.

**Comment by bsutton** [comment]
I've lost track of who the primary maintainer is?

Edit words

**Comment by seanbright** [comment]
That question is the impetus of this request. I’d rather the project not die and I have a lot of free time.

**Comment by bsutton** [comment]
I don't have any problem with you becoming a maintainer but don't want to
override any active maintainers.

Let's give it a couple of days and if there is no response I will look to
see if I still have the permissions to add you as a maintainer.

**Reply by seanbright** [reply]
Thoughts?

**Comment by bsutton** [comment]
So what permissions are you missing?

Do you have the maven keys?

**Reply by seanbright** [reply]
> So what permissions are you missing?

All of them? I don't have the ability to merge PRs or close issues.

> Do you have the maven keys?

I do not.

**Comment by bsutton** [comment]
Ok, I will start the process over  the weekend.

**Reply by seanbright** [reply]
I don't know who has ownership permissions to the asterisk-java organization but I would suggest they transfer ownership to you (or someone equally responsive), especially if they have no intention of being involved.

**Reply by seanbright** [reply]
Anything I can do to help get this process moving?

**Reply by scgm11** [reply]
maybe we can start adding as a contributor as my permissions allows me to merge

**Comment by piotrooo** [comment]
@seanbright and @scgm11, if you want to join the project, I maybe try to join as well (again). If you have time for a meeting, I can share my ideas about this project.

**Reply by piotrooo** [reply]
@seanbright, @scgm11 let me know if you are interested.

**Reply by seanbright** [reply]
Sure. I would like to get the outstanding issues resolved n the 3.x branch and then cut a new release ASAP. If I manually change the version to 3.40.0-SNAPSHOT in the POM does that present any issues for your release actions?

**Reply by seanbright** [reply]
Let's discuss [over here](https://github.com/asterisk-java/asterisk-java/discussions/487#discussioncomment-10463684).

**Comment by bsutton** [comment]
So my intent at this point is to hand over the project ownership to Sean
and then step out of the project.

The rest I will leave to you.
